runtime/vm/jvminit.c

```diff
@@ -7002,14 +7002,14 @@ freeClassNativeMemory(J9HookInterface** hook, UDATA eventNum, void* eventData, v
 {
 	J9VMClassUnloadEvent * data = eventData;
 	J9Class * clazz = data->clazz;
-	J9JavaVM *vm = data->currentThread->javaVM;
 	PORT_ACCESS_FROM_VMC(data->currentThread);
 
 	/* Free the ID table for this class, but do not free any of the IDs.  They will be freed by killing their
 		pools when the class loader is unloaded.
 	*/
 
 #if defined(J9VM_OPT_SNAPSHOTS)
+	J9JavaVM *vm = data->currentThread->javaVM;
```

Comment:
same here

Reply:
Fixed in https://github.com/eclipse/openj9/compare/74d9e544302db2e57a039f6bc3250808729d938a..a001f260c87796da222a7d5f0aa36ca27e7d5b69